docs/NooBaaNonContainerized/Health.md

```diff
@@ -0,0 +1,350 @@
+    - Type: String
+    - Enum: OK | NOTOK
+    - Description: Overall status of the system.
+
+- `memory`
+    - Type: String
+    - Description: Current memory being used by NooBaa service.
+
+- `error_code`
+    - Type: String
+    - Enum: See - [Health Errors](#health-errors)
+    - Description: Error code of the health issue uncovered by the Health CLI. Will be displayed in Health response if status is NOTOK.
+    
+
+- `error_message`
+    - Type: String
+    - Enum: See - [Health Errors](#health-errors)
+    - Description: Error message describing a health issue uncovered by the Health CLI. Will be displayed in Health response if status is NOTOK.
+
+- `service_status`
+    - Type: String
+    - Enum: "active" | "inactive" | "missing service status info"
+    - Description: NooBaa service status. Determined by whether the service is up and running.
+
+- `pid`
+    - Type: Number | "missing pid info"
+    - Description: NooBaa service process ID or Error message.
+
+- `endpoint_state`
+    - Type: { response: { response_code: "Running" | "NOT_RUNNING", response_message: String } }
+    - Description: NooBaa endpoint response.
+  
+- `total_fork_count`:
+    - Type: Number
+    - Description: Total number of expected forks in NooBaa.
+
+- `running_workers`
+    - Type: Array
+    - Description: Running NooBaa endpoint workers IDs.
+
+- `invalid_buckets`: 
+  - Type: [{"name": "bucket_name", "config_path": "/bucket/config/path" OR "storage_path": "/bucket/storage/path", "code": String }]
+  - Description: Array of invalid buckets, invalid buckets missing valid storage path or having invalid config JSON file.
+
+- `invalid_accounts`: 
+  - Type: [{ "name": "account_name", "config_path": "/account/config/path" OR "storage_path": "/path/to/accounts/new_buckets_path","code": String }] 
+  - Description: Array of invalid accounts, invalid account is account that - 
+    1. The account's new_buckets_path doesn't exist
+    2. The account doesn't have RW access to its new_buckets_path or having invalid config JSON file.
+
+- `valid_accounts`
+  - Type: [{ "name": account_name, "storage_path": "/path/to/accounts/new_buckets_path" }]
+  - Description: Array of all the valid accounts. If the all_account_details flag is set to true, valid_accounts will be included in the Health response.
+
+- `valid_buckets`
+  - Type: [{ "name": bucket_name, "storage_path": "/path/to/bucket/path" }]
+  - Description: Array of all the valid buckets. If the all_bucket_details flag is set to true, valid_buckets will be included in the Health response.
+
+- `error_type`
+  - Type: String
+  - Enum: 'PERSISTENT' | 'TEMPORARY'
+  - Description: For TEMPORARY error types, NooBaa attempts multiple retries before updating the status to reflect an error. Currently, TEMPORARY error types are only observed in checks for invalid NooBaa endpoints.
+ 
+
+## Example 
+```sh
+sudo node usr/local/noobaa-core/src/cmd/health --all_account_details --all_bucket_details
+```
+
+Output:
```

Comment:
On my output, I saw for example:
```json
"buckets_status":
{
    "invalid_buckets":[],
     "valid_buckets":[
          {"name":"bucket-1001-05","storage_path":"/tmp/nsfs_root1/my-bucket"}
     ],
    "error_type":"PERSISTENT"
}
```
Why do I see the error type near the buckets?